CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## [6.0.0](https://github.com/cleandart/react-dart/compare/5.7.1...6.0.0)
+
+This stable, __major__ release of react includes:
+
+### ReactJS 17.x Support
+
+The underlying `.js` files provided by this package are now ReactJS version `17.0.1`.
```

Comment:
I know we do in the linked PR, but can we also include a link to React 17 in the changelog?